Checkout/Checkout/Source/Model/Card+Scheme.swift

```diff
@@ -21,33 +21,33 @@ extension Card {
   ///   - Visa
   public enum Scheme: String, CaseIterable, CustomStringConvertible {
     case unknown
-    case visa
     case mada
+    case visa
     case mastercard
     case maestro
-    case amex
+    case americanExpress
     case discover
-    case diners
+    case dinersClub
     case jcb
 
     public init?(rawValue: String) {
       switch rawValue.lowercased() {
       case "unknown":
         self = .unknown
+      case "mada", "visa - mada", "mastercard - mada":
```

Comment:
after trimming white spaces
```suggestion
      case "mada", "visa-mada", "mastercard-mada":
```